src/net/udp/mod.rs

```diff
@@ -62,7 +63,7 @@ impl UdpSocket {
     /// calling `split` on the `UdpFramed` returned by this method, which will
     /// break them into separate objects, allowing them to interact more
     /// easily.
-    pub fn framed<C: UdpCodec>(self, codec: C) -> UdpFramed<C> {
+    pub fn framed<C: Encoder + Decoder>(self, codec: C) -> UdpFramed<C> {
```

Comment:
Could this fn be removed from `UdpSocket` in favor of `UdpFramed::new`? This change will happen on the TCP side as well, but probably not until the next release of tokio-io.

Reply:
`UdpFramed::new(socket, codec)`?

Reply:
Yes!

Reply:
Well, mb I should create another PR for this changes?

Reply:
Yeah, it can be done in another PR.